packages/components/src/Button.tsx

```diff
@@ -63,7 +63,7 @@ function getClassName(kind: ButtonKind, iconOnly: boolean): string {
     case 'ghost':
       return classNames('btn-link', {
         'btn-link-icon': iconOnly,
-        'px-2': iconOnly,
+        'custom-px-2': iconOnly,
```

Comment:
Can we just remove the `px-2` class entirely and then add `padding-left`/`padding-right` to the css for `btn-link-icon` instead?

Reply:
`btn-link-icon` already has its own `padding-left`/`padding-right` and a lot of non icon-only buttons use it as a class so I don't think its possible to simply move the `px-2` padding into `btn-link-icon` as that may alter the style of other buttons.

Reply:
You are correct. Maybe just name this class `btn-link-icon-only` then or something.

Follow up task: clean up `<button/>` uses to component `<Button/>`?

Reply:
There's also this in the base stylesheet

```
.btn-link-icon:after {
  padding-left: calc(100% - 2px);
}
```